docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -105,8 +105,8 @@ The following domain resource attributes are specific to Model in Image domains.
 | -------------------------                    |  ------------------ |
 | `domainHomeSourceType`                       |  Required. Set to `FromModel`. |
 | `domainHome`                                 |  Must reference an empty or non-existent directory within your image. Do not include the mount path of any persistent volume. Note that Model in Image recreates the domain home for a WebLogic pod every time the pod restarts.|
-| `configuration.model.configMap`             | Optional. Set if you have stored additional models in a config map as per [Optional WDT model config map](#3-optional-wdt-model-config-map). |
-| `configuration.secrets`                      | Optional. Set this array if your image or config map models contain macros that reference custom Kubernetes secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
+| `configuration.model.configMap`             | Optional. Set if you have stored additional models in a configmap as per [Optional WDT model configmap](#3-optional-wdt-model-config-map). |
+| `configuration.secrets`                      | Optional. Set this array if your image or configmap models contain macros that reference custom Kubernetes secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
```

Comment:
for line 154 (not sure why it won't let me add a comment there)  
i don't think this warning is quite accurate - they can always ask the dba to reset the password